src/components/ReviewsContainer/ButtonVote.jsx

```diff
@@ -1,9 +1,23 @@
 
-const ButtonVote = () => {
+import { useState } from 'react'
+import * as api from '../../api'
+
+const ButtonVote = ({review_id}) => {
+    
+    const [isError, setIsError] = useState(false)
+
+    function handleVote(vote){
+        api.patchVote(review_id, vote).then(() => {
```

Comment:
Were missing optimistic rendering, for the sake of user experience we would want to see the votes update immediately even on slow connections, if there was an error this would need to be reversed.